lib/src/value/string.ts

```diff
@@ -13,10 +13,19 @@ export class SassString extends Value {
   private readonly hasQuotesInternal: boolean;
 
   /** Creates a string with `text`, optionally with quotes. */
-  constructor(text: string, options?: {/** @default true */ quotes?: boolean}) {
+  constructor(
+    textOrOptions?: string | {quotes?: boolean},
+    options?: {quotes?: boolean}
+  ) {
```

Comment:
Let's make this an overload like SassList
```suggestion
  constructor(text: string, options?: ConstructorOptions);
  constructor(options?: ConstructorOptions);
  constructor(
    textOrOptions?: string | ConstructorOptions,
    options?: ConstructorOptions
  ) {
```